specifyweb/frontend/js_src/lib/components/DataModel/saveBlockers.tsx

```diff
@@ -131,12 +154,34 @@ export function getFieldBlockers(
   resource: SpecifyResource<AnySchema>,
   field: LiteralField | Relationship
 ): RA<string> {
-  const blockers = saveBlockers.get(resource)?.blockers ?? [];
+  const blockers = getResourceBlockers(resource).blockers;
   return blockers
     .filter((blocker) => blocker.field === field)
     .map(({ message }) => message);
 }
 
+function usePreviouslySetBlocker(
```

Comment:
this is not a react hook, so please don't name it as `use*`
quite confusing)
(and ESLint might get confused too)